src/open_producten/producten/admin/product.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+
+from open_producten.producten.models import Product
+
+
+@admin.register(Product)
+class ProductAdmin(admin.ModelAdmin):
+    list_display = (
+        "product_type_name",
+        "aanmaak_datum",
+        "start_datum",
+        "eind_datum",
+        "bsn",
+        "kvk",
+    )
+    list_filter = ("product_type__naam", "aanmaak_datum", "start_datum", "eind_datum")
+    autocomplete_fields = ("product_type",)
+    search_fields = ("product_type__naam",)
+
+    @admin.display(description="Product Type")
+    def product_type_name(self, obj):
+        return obj.product_type.name
```

Comment:
```suggestion
    @admin.display(description="Product Type")
    def product_type_name(self, obj):
        return obj.product_type.naam
```

Not called name anymore, admin listview will crash if an object exists